### ISSUE TITLE
[BUG] karma-accessibility-checker double-parses `archives.json`

### ISSUE THREAD

**Issue by asudoh** [IBMa/equal-access#104]
## What tool(s) are you using?

- [ ] `accessibility-checker-engine`
- [ ] `accessibility-checker`
- [x] `karma-accessibility-checker`
- [ ] `rule-server`
- [ ] `boilerplates`
- [ ] `other`

## Detailed description

> Node version:

12.16.1

> Platform: 

macOS

> Describe in detail the issue you're having.

Got the following error from Karma with `karma-accessibility-checker` set up:

```
02 06 2020 15:11:11.931:ERROR [karma-server]: Server start failed on port 9876: SyntaxError: Unexpected token o in JSON at position 1
```

Making a change to the code emitting the log above so it emits `error.stack` yielded to the following:

```
02 06 2020 15:15:56.798:ERROR [karma-server]: Server start failed on port 9876: SyntaxError: Unexpected token o in JSON at position 1
    at JSON.parse (<anonymous>)
    at Object.processACConfig (/Users/asudoh/git/ibm-dotcom-library/node_modules/karma-accessibility-checker/lib/ACCommon.js:344:34)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Object.processKarmaConfiguration (/Users/asudoh/git/ibm-dotcom-library/node_modules/karma-accessibility-checker/lib/ACCommon.js:586:20)
    at async ACEngineLoaderAndConfig (/Users/asudoh/git/ibm-dotcom-library/node_modules/karma-accessibility-checker/lib/ACEngineLoader.js:70:5)
```

So the stack trace seems to point to https://github.com/IBMa/equal-access/blob/2d23251/karma-accessibility-checker/src/lib/ACCommon.js#L339, and found that it attempts to run `JSON.parse()` against an object that is parsed (already) at https://github.com/IBMa/equal-access/blob/2d23251/karma-accessibility-checker/src/lib/ACCommon.js#L335, and thus the 2nd `JSON.parse()` call is effectively parsing `[object Object]`).

## Steps to reproduce the issue

I can imagine it happens any `karma-accessibility-checker` setup, but please contact me if you need any more info.
